okex-client/src/client/primitives.rs

```diff
@@ -147,4 +147,6 @@ pub struct OkexClientConfig {
     pub simulated: bool,
     pub position_mode: OkexPositionMode,
     pub position_side: OkexPositionSide,
+    pub order_type: OkexOrderType,
```

Comment:
I find it very confusing to add these settings here.

- are they static? ie will they be the same on every call to `place_order` or `close_position`? If so we should not expose them as config options but hard code it somewhere.
- might they change from one call to `place_order` to the next? If so then they should be passed as function arguments.

Reply:
I had a chat with @sebastienverreault about these and he provided some clarification. 

1. For the `position_side`, it has to be `net`, given that the `position_mode` is `net`. The OKX documentation highlights it [here](https://www.okx.com/docs-v5/en/#rest-api-trade-place-order) (check `posSide` ).
2. For the `order_type`, it's `market` that we use. 

I don't think that these will change except we change our strategy. Given that some of these config parameters are shared in the call to `place_order` and `close_position`, it seemed better to create them once (in the config).

Reply:
We probably have 2 types of config. One that we input at runtime and one that remains static / hard coded.